src/components/DocumentListItem/DocumentListItem.test.js

```diff
@@ -15,33 +15,33 @@ describe("DocumentListItem component", () => {
   it("renders the name of the document", () => {
     var document = createTestDocument({ name: "Document Name" });
 
-    const { getByTestId } = render(<DocumentListItem document={document} />);
+    const { getByText } = render(<DocumentListItem document={document} />);
```

Comment:
The changes below are just a refactor. Shows how `getByText` doesn't require as much setup and is less brittle. Though it's also less specific. So a tradeoff.

Reply:
We decided we like the specificity for now